inc/healthcheck/cavalcade/namespace.php

```diff
@@ -51,10 +51,13 @@ function set_last_run() {
  * @return boolean|WP_Error True if healthy, error otherwise.
  */
 function check_health() {
+	switch_to_blog( 1 );
```

Comment:
Hmm I'm not sure if `1` is always the case, I think this should use whatever is checked to actually enqueue the job in the first place.

Reply:
This seem to work?

Reply:
What if the main site ID isn't `1` though?

Reply:
I did change it last night, please see the latest commit @joehoyle ?